### ISSUE TITLE
Add another theme

### ISSUE THREAD

**Issue by potherca** [potherca/ApacheDirectoryListTheming#1]
It would be nice to have an other theme than the current Firefox 3 based on.
Something similar to Github's could be nice, for a more similar feel between repo and local browsing?

Depends on #9

**Comment by potherca** [comment]
Theming has already been moved to a separate directory in bc84cb96.

**Comment by potherca** [comment]
Fixed in [release 0.2.0](https://github.com/potherca/ApacheDirectoryListTheming/releases/tag/0.2.0)
